src/redisraft.h

```diff
@@ -809,11 +805,15 @@ void handleFsyncCompleted(void *arg);
 /* util.c */
 int RedisModuleStringToInt(RedisModuleString *str, int *value);
 char *catsnprintf(char *strbuf, size_t *strbuf_len, const char *fmt, ...) __attribute__((format(printf, 3, 4)));
+int safesnprintf(void *buf, size_t size, const char *fmt, ...) __attribute__((format(printf, 3, 4)));
 void AddBasicLocalShardGroup(RedisRaftCtx *rr);
 void FreeImportKeys(ImportKeys *target);
 unsigned int keyHashSlot(const char *key, size_t keylen);
 unsigned int keyHashSlotRedisString(RedisModuleString *s);
 RRStatus parseHashSlots(char *slots, char *string);
+bool parseLongLong(const char *str, char **end, long long *val);
```

Comment:
Better to have all these util methods in a separate `util.h` header, but not urgent for now, only for future clean up.

Reply:
I use separate header for new stuff, e.g metadata.h/.c 

For the existing code, we still need to establish a coding convention (not limited for separate header file, we need to agree on bunch of other things imho). I tried to write a doc for it to discuss but need to finalize it. After that, I'd like to solve these kind of issues forever with a quick refactor PRs.

Reply:
Sure. As I mentioned - not an urgent one.